docs/content/2.deploy/providers/edgio.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Edgio
+description: 'Discover Edgio (formerly Layer0) preset for Nitro!'
+---
+
+**Preset:** `edgio` ([switch to this preset](/deploy/#changing-the-deployment-preset))
+
+Nitro provides a built-in preset to generate output format compatible with [Edgio (formerly Layer0)](https://edg.io/).
+
+Edgio (formerly Layer0) extends the capabilities of a traditional CDN by not only hosting your static content, but also providing server-side rendering for progressive web applications as well as caching both your APIs and HTML at the network edge to provide your users with the fastest browsing experience.
+
+If this is your first time deploying to Edgio, the interactive CLI as part of the `deploy` command will prompt to authenticate using your browser. You may also [sign up](https://app.layer0.co/signup) prior to deployment.
+
+## Install the Edgio CLI
+
+```bash
+npm i -g @edgio/cli
+```
+
+## Initialize your project for use with Edgio
+
+```bash
+edgio init --connector=@edgio/nitropack
+```
+
+## Testing locally
+
+You can use Edgio to test your app's developement experience locally:
+
+```bash
+edgio dev
+```
+
+Under the hood, `edgio dev` runs `npx nitropack dev` on the port 3001.
```

Comment:
I prefer to advice `nuxt`/`nitro` CLI primarily for development for consistency (unless there is a specific feature you want in dev mode, would be happy to discuss how we can implement it as cli/framework extension)